src/manifold/src/manifold.cpp

```diff
@@ -775,4 +778,61 @@ Manifold Manifold::TrimByPlane(glm::vec3 normal, float originOffset) const {
 }
 
 ExecutionParams& ManifoldParams() { return params; }
-}  // namespace manifold
\ No newline at end of file
+
+Manifold Manifold::Hull(const std::vector<glm::vec3>& pts,
+                        const std::vector<float>& props, const int numProp) {
+  const int numVert = pts.size();
+  // FIXME: Should there be a hull specific error?
```

Comment:
Sure, I suppose we can add an error status for this.

Reply:
Partly as a reminder to myself, I've forgotten to check so far what this QuickHull returns if all points are coplanar. However it's found though, will have to have that result in an error Manifold as well.

Reply:
So, `getConvexHull` returns 4 triangles when a coplanar rectangle of vertices is given to it, but after passing through the `Manifold(mesh)` constructor, the result is an empty `Manifold` (`NoError`). Should we just return an empty Manifold in both cases, have an error for one and empty for the other, or an error for both?

If error(s):
- `AllVerticesCoplanar` (or simply `NotManifold`)
- `TooFewVertices` (which also technically is coplanar, and not manifold)

Reply:
Yeah, let's prefer no error if there isn't a huge reason to have one.